package/com.unity.formats.usd/Samples/UsdTimelinePlayable/README.md

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Google Inc. All rights reserved.
```

Comment:
New file has the 'Google' copyright

Reply:
I think the Google Copyright is from when Jeremy was at Google - I believe he started this project when he was part of Google and i suspect he copy-pasta'd some of his scripts